README.md

```diff
@@ -1,4 +1,4 @@
-![Formula CI](https://github.com/SUSE/drbd-formula/drbd-formula/workflows/Formula%20CI/badge.svg)
+[![Formula CI](https://github.com/SUSE/drbd-formula/actions/workflows/drbd-formula-ci.yml/badge.svg)](https://github.com/SUSEyy/drbd-formula/actions/workflows/drbd-formula-ci.yml)
```

Comment:
Thanks for the commit.
Maybe there is a typo in the URL which direct to a not existed page. (I guess s/SUSEyy/SUSE/)

Reply:
Yes, thank you :+1:   I corrected the typo.  The funny thing is it was displaying the badge correct on my fork.